services/src/messaging/src/checkstyle/checkstyle-suppress-known-issues.xml

```diff
@@ -274,6 +274,7 @@
     <suppress files="src/main/java/org/openkilda/messaging/command/switches/SwitchRulesValidateRequest.java" lines="21" checks="CustomImportOrder"/>
     <suppress files="src/main/java/org/openkilda/messaging/command/switches/SwitchRulesValidateRequest.java" lines="30" checks="Indentation"/>
     <suppress files="src/main/java/org/openkilda/messaging/command/switches/SwitchRulesValidateRequest.java" lines="31" checks="Indentation"/>
+    <suppress files="src/main/java/org/openkilda/messaging/command/switches/PortConfigurationRequest.java" lines="22" checks="CustomImportOrder"/>
```

Comment:
???

Reply:
+1
It was agreed that checkstyle-suppress-known-issues.xml files are used to skip the checkstyle violations that were introduced prior to checkstyle enforcement.